makefile

```diff
@@ -80,6 +80,10 @@ DEBUG_SET_ENV_VARS := \
 	export FEATURE_PROTOTYPE_HEADER_FOOTER_ENABLED=true; \
 	export FEATURE_CAMPAIGN_PAGES_ENABLED=true; \
 	export FIND_A_SUPPLIER_CONTACT_URL=http://supplier.trade.great:8005/industries/contact/
+	export EU_EXIT_ZENDESK_SUBDOMAIN=debug; \
+	export EUEXIT_AGENT_EMAIL=test@example.com; \
+	export FIND_A_SUPPLIER_CONTACT_URL=http://supplier.trade.great:8005/industries/contact/; \
+	export FEATURE_EXPORT_JOURNEY_ENABLED=true
```

Comment:
please put `EU_EXIT_ZENDESK_SUBDOMAIN=debug` in your conf/.env` instead

`EUEXIT_AGENT_EMAIL` is no longer needed in master